README.md

```diff
@@ -73,6 +73,53 @@ The AET dictionary JSON can also be specified directly via the "--aet_dictionary
 
 For the list of command line flags, see [here](import/src/main/java/com/google/cloud/healthcare/imaging/dicomadapter/Flags.java)
 
+## Routing to multiple DICOM stores
```

Comment:
Since the readme is getting a bit long, and this feature is likely only for advanced users, let's move this section to the wiki and add a link here instead. Something like:
"The Import Adapter supports routing to multiple DICOM stores based on tags in the incoming instance and the client application entity title. Please see \[this page\](link_to_wiki) for more information."

Reply:
Ok, extracted to wiki.